StreamChatSwiftUI.xcodeproj/project.pbxproj

```diff
@@ -1742,6 +1742,7 @@
 /* Begin PBXShellScriptBuildPhase section */
 		8465FCCE27468DFB00AF091E /* SwiftGen */ = {
 			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
```

Comment:
What is this?

Reply:
It makes `SwiftGen` run on every build. This is basically how it worked, but Xcode 14 started to show the warning that we need to mark it as `alwaysOutOfDate` or set up the conditions when to run the phase.